lib/engine/config/game/g_1835.rb

```diff
@@ -197,15 +197,24 @@ module G1835
           ],
           "free": true,
           "count": 1
-        }
+        },
```

Comment:
Formatting seems a bit off, here and below.

Reply:
Please fix the probably tabs/space issue, then it's good to go.